webapp/components/Navbar/Navbar.spec.js

```diff
@@ -0,0 +1,35 @@
+import { mount } from '@vue/test-utils'
+import Navbar from './Navbar'
+
+describe('Given a navbar', () => {
+  let wrapper
+  beforeEach(() => {
+    wrapper = mount(Navbar)
+  })
+  it('render buttons', () => {
+    expect(wrapper.text()).toContain('Login')
```

Comment:
⭐ ⭐ For a menu component which shows a login or logout button and its software tests.